README.md

```diff
@@ -222,6 +197,61 @@ asynchronous response parts, respond to the client and return the `Plug.Conn`.
 `:status_callbacks` must only be given when `:response_mode` is `:stream`,
 which is the default.
 
+## Usage in Phoenix
```

Comment:
No changes of content in this section. Only moved it down and went from `###` to `##` so that it appears on the sidebar in Hexdocs.pm.